doc-content/enterprise-only/cases/case-management-creating-manager-approval-proc.adoc

```diff
@@ -2,88 +2,92 @@
 = Creating the Manager approval business process
 The manager approval process determines whether or not the order will be placed or rejected.
 
-.Business Process (legacy) editor
-image::cases/legacy-editor-nav.png[Editor navigation]
-
-*1*: The canvas where you build your process diagram. Drag elements from the *Object Library* to the canvas to design your process.
-
-*2*: The *Object Library*.
-
-*3*: The *Properties* panel displays the properties of the selected element. If no element is selected, the panel contains process properties.
-
 .Procedure
-. In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* -> *IT_Orders_New* -> *orderhardware Business Process (legacy)*.
-. In the process designer, expand *Tasks* in the *Object Library* drag a new *User* task on to the canvas.
-. Click the new user, click image:cases/left-arrows.png[] to open the *Properties* panel, and input the following values:
-* *Groups* - `supplier`
-* *Name* - `Prepare hardware spec`
-* *Task Name* - `PrepareHardwareSpec`
-* *Description* - `Prepare hardware specification for #{initiator} (order number #{CaseId})`
-
-. Click the down arrow in the *Assignments* field, add the following new conditions in the *Prepare hardware spec Data I/O* window. For the first output assignment, select *Custom* for the *Data Type* and input `org.jbpm.document.Document`. Click *Save*.
+. In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* -> *IT_Orders_New* -> *orderhardware Business Processes*.
+. Create and configure the *Prepare hardware spec* user task:
+.. Drag a start event on to the canvas and create an outgoing connection from the start event to a task and convert the new task to a user task.
```

Comment:
on to - onto. Search for.

Reply:
Searched and destroyed accordingly. Thanks for the catch Eagle eye